.github/workflows/package-helm.yaml

```diff
@@ -30,10 +38,68 @@ jobs:
             mv $SRC_FILE $DEST_FILE
           done
 
-      - name: Upload Helm packages to GitHub releases
+      - name: Upload Stable Helm Charts to GitHub Release
         uses: softprops/action-gh-release@v1
         with:
           name: ${{ github.ref_name }}
           draft: true
           append_body: true
           files: nri-*helm-chart*.tgz
+
+  unstable:
+    concurrency:
+      group: unstable-helm-charts
+      cancel-in-progress: false
+    if: ${{ github.env.IS_RELEASE != 'true' }}
+    permissions:
+      packages: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Deep Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install Helm
+        uses: azure/setup-helm@v4.0.0
+
+      - name: Package Unstable Helm Charts
+        id: package-charts
+        run: |
+          # For unstable chart verison we use:
+          #   - chart version: latest tag+1 patch + '-unstable'
+          #   - image version: 'unstable'.
+          majminpatch="$(git describe | sed -E 's/v([0-9]*\.[0-9]*\.[0-9]*).*$/\1/')"
+          majmin=${majminpatch%.*}
+          patch=${majminpatch##*.}
+          CHART_VERSION="${majmin}.$((patch+1))-unstable"
```

Comment:
Hmm, what do we get for the main branch? We get `v0.4.1-unstable` when even `v0.4.0` is not released, yet?

Also, another question I'm pondering whether it would make sense to just have per-majmin or per-release-branch unstable chart versions like `v0.4-unstable`.

I think this would make it cleaner in the future if we enable unstable chart for release branches. E.g. it would keep the registry a bit cleaner and versions like `v0.4.1-unstable` virtually becomes junk/noise when v0.4.1 is out.

Reply:
> Also, another question I'm pondering whether it would make sense to just have per-majmin or per-release-branch unstable chart versions like v0.4-unstable.
>
> I think this would make it cleaner in the future if we enable unstable chart for release branches. E.g. it would keep the registry a bit cleaner and versions like v0.4.1-unstable virtually becomes junk/noise when v0.4.1 is out.

True. It makes much more sense to have per-majmin unstable version.

Reply:
Updated to use per-majmin versions.